bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsMagnitudeTable.scala

```diff
@@ -103,11 +103,16 @@ object GemsMagnitudeTable extends MagnitudeTable {
   }
 
   lazy val CanopusWfsMagnitudeLimitsCalculator = new CanopusWfsCalculator {
-    override def gemsMagnitudeConstraint(starType: GemsGuideStarType, nirBand: Option[MagnitudeBand]) =
-      magLimits(RBandsList, 16.3, 8.8)
+    // The values provided by science are assumed to be at SB ANY, CC50, and IQ70.
+    // Since SB ANY results in an adjustment of -0.5 mag, we add 0.5 mag to account for this.
+    private val FaintLimit  = 17.5
+    private val BrightLimit = 11.0
```

Comment:
So the values they actually specified are (10.5, 17.0), which for our reference conditions of SB50, CC50, IQ70 would translate to (11.0,17.5)?  Is that correct?  It's surprising that they give values in a different set of reference conditions than in the past.

A comment about where the adjustment is defined might be good (package object `edu.gemini.catalog.api`) -- I mean to piggy pack on a future PR.

Reply:
Yes, that is correct. I agree that it is odd and I'm not sure why this is the case, but it was confirmed by Rodrigo in the NGS2 OT document.

Agreed on the comment. I'll make that in the PR I'm working on now regarding the MASCOT changes.